src/IBM.WatsonDeveloperCloud.SpeechToText.v1/SpeechToTextService.cs

```diff
@@ -255,49 +266,70 @@ public SpeechModels ListModels(Dictionary<string, object> customData = null)
+        /// Spanish transcription only. See [Smart
+        /// formatting](https://console.bluemix.net/docs/services/speech-to-text/output.html#smart_formatting).
+        /// (optional, default to false)</param>
         /// <param name="speakerLabels">If `true`, the response includes labels that identify which words were spoken by
         /// which participants in a multi-person exchange. By default, no speaker labels are returned. Setting
         /// `speaker_labels` to `true` forces the `timestamps` parameter to be `true`, regardless of whether you specify
-        /// `false` for the parameter.
-        ///
-        ///  To determine whether a language model supports speaker labels, use the **Get models** method and check that
-        /// the attribute `speaker_labels` is set to `true`. You can also refer to [Speaker
+        /// `false` for the parameter. To determine whether a language model supports speaker labels, use the **Get
+        /// models** method and check that the attribute `speaker_labels` is set to `true`. See [Speaker
         /// labels](https://console.bluemix.net/docs/services/speech-to-text/output.html#speaker_labels). (optional,
         /// default to false)</param>
+        /// <param name="customizationId">**Deprecated.** Use the `language_customization_id` parameter to specify the
+        /// customization ID (GUID) of a custom language model that is to be used with the recognition request. Do not
+        /// specify both parameters with a request. (optional)</param>
         /// <param name="customData">Custom data object to pass data including custom request headers.</param>
         /// <returns><see cref="SpeechRecognitionResults" />SpeechRecognitionResults</returns>
-        public SpeechRecognitionResults RecognizeSessionless(byte[] audio, string contentType, string model = null, string customizationId = null, string acousticCustomizationId = null, string baseModelVersion = null, double? customizationWeight = null, long? inactivityTimeout = null, List<string> keywords = null, float? keywordsThreshold = null, long? maxAlternatives = null, float? wordAlternativesThreshold = null, bool? wordConfidence = null, bool? timestamps = null, bool? profanityFilter = null, bool? smartFormatting = null, bool? speakerLabels = null, Dictionary<string, object> customData = null)
+        public SpeechRecognitionResults RecognizeSessionless(byte[] audio, string contentType, string model = null, string languageCustomizationId = null, string acousticCustomizationId = null, string baseModelVersion = null, double? customizationWeight = null, long? inactivityTimeout = null, List<string> keywords = null, float? keywordsThreshold = null, long? maxAlternatives = null, float? wordAlternativesThreshold = null, bool? wordConfidence = null, bool? timestamps = null, bool? profanityFilter = null, bool? smartFormatting = null, bool? speakerLabels = null, string customizationId = null, Dictionary<string, object> customData = null)
```

Comment:
I think the rename from `customizationId` to `languageCustomizationId` is going to need a handwritten change to avoid breaking users. If they're passing arguments positionally, they'll be fine, but if they're using a named parameter, this change will force them to change their code.

Reply:
Ahh no you're completely right! That's a huge relief. Everything looks good then 👍